cloudinary-core/src/main/java/com/cloudinary/Api.java

```diff
@@ -519,6 +519,11 @@ public ApiResponse updateResourcesAccessModeByTag(String accessMode, String tag,
         return updateResourcesAccessMode(accessMode, "tag", tag, options);
     }
 
+    public ApiResponse deleteFolder(String folder, Map options) throws Exception {
```

Comment:
@nitzanj , please add a javadoc and explain how the 'folder' string supposed to look like